scripts/ci/build-installers.sh

```diff
@@ -48,8 +48,8 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$TARGET_ARCH" \
     -s "$(pwd)" \
-    -c 'make installers-all'
+    -c 'make electron-develop installers-all'
```

Comment:
Thanks for the explanations Juanchi, I think I understand what's going on now, so I'll approve :)

It's super-hacky, but could another solution be to have a `etcher-shrinkwrap.json` and a `electron-builder-shrinkwrap.json` and then copy each file to `npm-shrinkwrap.json` as appropriate? (and I guess you could rename the `node_modules` directory back and forth too). But of course that may well be worse than the current situation ;-)